pageserver/src/import_datadir.rs

```diff
@@ -456,6 +455,7 @@ pub async fn import_wal_from_tar(
             if let Some((lsn, recdata)) = waldecoder.poll_decode()? {
                 let mut decoded = DecodedWALRecord::default();
                 decode_wal_record(recdata, &mut decoded, tline.pg_version)?;
+                // let (ephemeral_file_ready_buf, special_records) = decode_wal_record(recdata, tline.pg_version);
```

Comment:
rm commented code?

Reply:
oops. Done in https://github.com/neondatabase/neon/pull/9504/commits/c825b2aafa3ee22f21d63cbfef2cdfb89016dddd